src/core/CMakeLists.txt

```diff
@@ -45,6 +52,21 @@ target_compile_definitions(openthread-radio PRIVATE
     OPENTHREAD_RADIO=1
 )
 
+target_compile_options(openthread-ftd PRIVATE
+    ${OT_CFLAGS}
+    ${OT_CXXFLAGS}
```

Comment:
Potentially might be worth combining ``OT_CFLAGS`` and ``OT_CXXFLAGS`` into one ``OT_FLAGS`` just to reduce boilerplate, but no strong feelings.